### ISSUE TITLE
v3.0.10 source tarball is missing variant.hpp

### ISSUE THREAD

**Issue by springmeyer** [mapnik/mapnik#3334]
Somehow when I packaged the tarball the variant folder as added but not the code itself. Will work to fix asap:

![screen shot 2016-02-29 at 11 50 19 am](https://cloud.githubusercontent.com/assets/20300/13406883/b08071f8-deda-11e5-9199-8f9d10b9a2ed.png)

**Comment by tomhughes** [comment]
It's in the one I downloaded (from http://s3.amazonaws.com/mapnik/dist/v3.0.10/mapnik-v3.0.10.tar.bz2) on Saturday morning:

```
bericote [~/src/fedora/mapnik] % tar jtvf mapnik-v3.0.10.tar.bz2 | fgrep variant.hpp
-rw-r--r-- dane/wheel     2766 2016-02-26 02:02 mapnik-v3.0.10/include/mapnik/util/variant.hpp
-rw-r--r-- dane/wheel    27628 2016-02-26 02:02 mapnik-v3.0.10/deps/mapbox/variant/variant.hpp
```

**Comment by springmeyer** [comment]
@tomhughes life safer - thank you. Your comment helped me realized I was confused. What is actually happening is that:

  - I tried testing homebrew with latest v3.0.10
  - and therefore updated the formula link from v3.0.9 to v3.0.10 (https://github.com/Homebrew/homebrew/blob/ae41d428b82de8c615e1a11883eb7995d489ae74/Library/Formula/mapnik.rb#L4)
  - and that tarball does not contain the variant.hpp.

But of course it does not because that is the auto-generated `tar.gz` from github not our manually created `tar.bz2`.

**Comment by springmeyer** [comment]
So, false alarm. The fix will be either to have the homebrew formula download the right tarball (not the github autogenerated one) or to actually fix #3297.

**Comment by springmeyer** [comment]
refs https://github.com/Homebrew/homebrew/pull/49407
